docs/user-guide/global-configurations/deployment-window.md

```diff
@@ -23,7 +25,7 @@ Technically, both of them are different methods of restricting deployments to an
 
 ## Configuring Deployment Window
 
-(This is configured by super-admins. Details yet to be added.)
+(This is configured by super-admins using APIs)
```

Comment:
Can we finish this up with devtctl commands, btw we also have some progress on FE of Global configuration part of deployment window. I believe we can close this PR asap now

Reply:
UI config added now, hence marking this as resolved.